packaged/lisk.sh

```diff
@@ -19,6 +19,9 @@
 
 # shellcheck disable=SC2129
 
+set -eo pipefail
+IFS=$'\n\t'
+
 cd "$(cd -P -- "$(dirname -- "$0")" && pwd -P)" || exit 2
```

Comment:
we could switch to using `cd $( cd -P -- "$( dirname -- "${BASH_SOURCE[0]}" )" && pwd -P )` (just like we do in [env.sh](https://github.com/LiskHQ/lisk-scripts/blob/17de1a2ad2e8b06df03688defca1ecce74e43bee/packaged/env.sh#L20))
also note that the `|| exit 2` since we now use `set -e`